modules/ROOT/pages/dashboards/advanced-features/filters-and-parameters.adoc

```diff
@@ -0,0 +1,87 @@
+= Filters and parameters
+:description: Use parameters in your Aura dashboards to further customize queries or use them in filters for node and relationship properties.
```

Comment:
We should change 'Aura dashboards' --> 'Dashboards', so that the docs can be reused for Desktop later.